### PR TITLE
Updated github workflow document

### PR DESCRIPTION
**Pull request by DeepBlockDeepak**
Hey, @ntjohnson1 . What do you think?

### CODE REVIEW

File: WORKFLOW.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+If you want to propose a change to [Python Tensor Toolbox](https://github.com/sandialabs/pyttb), follow the steps below:
+
+1. **Create an Issue**
+    - Use the [Issues tab](https://github.com/sandialabs/pyttb/issues) of th pyttb GitHub repository to propose a change. 
+    - See GitHub's guide on [Creating an issue](https://github.com/features/issues).
+
+
+1. **Fork the Repository and Create a New Branch**
+    - To make changes, you'll need to fork the pyttb repository and create a new branch in your fork. 
+    - See GitHub's [Getting Started Guide](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
```
Comment (ntjohnson1) [inline WORKFLOW.md:14]: NIT: Everything else ends with a period.

```suggestion
    - See GitHub's [Getting Started Guide](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
```

**Review by ntjohnson1 — CHANGES_REQUESTED** [review]
A couple of minor things but otherwise looks good to me. As I mentioned on the previous review I would lean towards dropping all the external links but I could see including them being more user friendly to subject matter experts who don't spend as much time on github.

1. Not super critical here but good to err on the side of a more verbose PR description in case someone is back tracking through the history in the future. We have the link to the issue with lots of discussion, but a summary here to avoid the need to click into that is usually appreciate when I end up putting on my archeologist hat for something.
2. Left a nit about the period.
3. This whole thing should be appended as an additional sub-section to the `CONTRIBUTING.md` to simplify needing to look around. Basically that one page should be enough for someone to figure out how to submit to the repo https://github.com/sandialabs/pyttb/contribute

**Comment by DeepBlockDeepak** [comment]
I got rid of the external links, and appended the slightly revised file to the end of CONTRIBUTING.md. I'm a bit fuzzy on the forking process but I think that the details are accurate.

**Review by dmdunla — APPROVED** [review]

**Review by dmdunla — APPROVED** [review]

**Review by ntjohnson1 — APPROVED** [review]

**Comment by ntjohnson1** [comment]
> I'm a bit fuzzy on the forking process but I think that the details are accurate.

Just for your own experience it might be worth opening a PR for some small change at some point via the fork and submit approach. That's how I do all of my work here if you want to look at any of my closed PRs for an example.

**Pull request merged**